File: 100-Days-Series/Day 87/PerfectNumber.js

// Arrow Function to print perfect number Square between 1 to N
const getPrintPerfectSquareNumber=(n)=>{
    for(let i=1;i*i<=n;i++){
        console.log((i*i));
    }
}
// Let's Start Here
let n=20;
console.log("Perfect Square Number between 1 and N :");
getPrintPerfectSquareNumber(n);

File: 100-Days-Series/Day 87/PerfectNumber.java
public class PerfectNumber {
    // Method to print perfect number Square between 1 to N
    public static void getPrintPerfectSquareNumber(int n){
        for(int i=1;i*i<=n;i++){
            System.out.print((i*i)+" ");
        }
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start Here
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the value of N : ");
        int  n=sc.nextInt();
        System.out.println("Perfect Square Number between 1 and "+n+"+ :\n");
        getPrintPerfectSquareNumber(n);
    }
}
